

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<html>
<head>
<style>

</style>
   <title> Mohsen Botlani's Homepage </title>
</head>
<body> <table align="center" border="0" width="1200px"><tbody><tr><td>
   <table border=0 style="width=180%">
       <tr >
	   <th>
	      <td width = "180" >
		     <img src="Figures/mohsenimage.png"  style="width:170px;height:auto">
		  </td>
		  <td width="5%"></td>
          <td valign="top" align="left" >
	  	     <h2> <i>Mohsen Botlani</i> </h2>
			 Ph.D.[<a href="MohsenBotlaniResume.pdf">Resume</a>] 
			 <br>
			 <a href="http://biology.usf.edu/cmmb/" target="_blank" > Cell Biology, Microbiology and Molecular Biology</a>
			 <br>
			 <a href="http://labs.cas.usf.edu/cbb/"><i>Computational Biophysics Lab</i></a>
			 <br>
			 University of South Florida
			 <br>
			 4202 East Fowler Avenue
			 <br>
			 Tampa, FL, 33620
			 <br>
			 <!-- <br>
			 <b>Office:</b> ENB 215 -->
			 <br>
			 <b>Email:</b> mohsenb [at] mail.usf.edu
			 <br>
             <b>Email:</b> m.botlani [at] gmail.com			 
	      </td>
          <td valign="top" >
		     <!--Home <br>
             Teaching <br>
             Research <br>
             Service <br>
             Vita <br> -->
		  </td>
		  </th>
		  
		  <th>
		  <img src="Figures/4ird.png"  style="width:325px;height:auto">
		  </th>
		  
       </tr>
   </table>
   		   	 <hr>   		 
   	         <h2> About Me </h2>
   			
			<p style="font-size: 13pt;"> <blockquote>I obtained my Ph.D. from the Department of Cell Biology, 
			Microbiology and Molecular Biology, University of South Florida under supervision of 
			Dr. <a href="http://biology.usf.edu/cmmb/faculty/svarma/">Sameer Varma</a>. 
			My PhD research lied at the intersection of machine learning and computational biology. 
			Broadly, I applied advanced machine learning methods to model protein regulations.
			I received my undergraduate degree in Material Sicnece and Engineering from <a href="http://www.iut.ac.ir/en/">Isfahan University of Technology.</a></blockquote>
   			 <br>
			 <hr>
<!--    		     <h2> <font color="red"> News </font> </h2>
   			 <p style="font-size: 13pt;"> Our paper entitled as <b>"An Incentive Mechanism for Crowdsensing Markets with Multiple Crowdsourcers"</b> got accepted in IEEE Internet of Things Journal. 
   			 <br>
 			 <hr>
 -->			 
   	         <h2> Research Interests </h2>
   			 <ul>
                <li> Computational Biology, Bioinformatics, System Biology, Omics Data Analysis, Pharmacogenomics, Computational Oncology, Health informatics </li>
                <li> Machine Learning: Artificial Neural Networks/Deep Learning, Kernel methods </li>
				<li> Big Data Analytics, Statistical Modeling </li>
                <li> Probabilistic Modelling, Bayesian Statistics </li>
				<li> Network Analysis/Graph Theory </li>
				<li> Evolutionary Game Theory, Algorithmic Game Theory </li>
             </ul>
			 
			  <hr>
   		     

			 
			 <!-- <hr>     -->
   		     <h2> Work and Research Experiences </h2>
			 
				<table style="font-size: 11pt;" border="0" width="60%">
	
	
                <tbody><tr>

		<td width="170"><img src="Figures/m2genLogo.png" style="width:120px;height:80"></td>

		<td>

			<p style="font-size: 15pt;"><b>Bioinformatics Staff Scientist at <a href="http://m2gen.com/">M2Gen</a></b> <br>
		        
		<hr>
		
		<tbody><tr>
		
		
		<tbody><tr>

		<td width="170"><img src="Figures/wellcareLogo.gif" style="width:120px;height:100"></td>

		<td>

			<p style="font-size: 15pt;"><b>Data Scientist at <a href="https://www.wellcare.com/">WellCare</a></b> <br>
		        <p style="font-size: 13pt;"><i><b> Customer retention predictive model development by usuing machine learning methods. </a> </i></b><br>
		        <p style="font-size: 13pt;"><i><b>Development of all-cause mortality rate predictive models of members based on their age, 
				gender and common laboratory tests by machine learning methods.</a> </i></b><br>
			 
		<hr>
		
		<tbody><tr>
		

		<tbody><tr>

		<td width="170"><img src="Figures/moffitt-cancer-center-logo.png" style="width:100px;height:80"></td>

		<td>

			<p style="font-size: 15pt;"><b>Research Assistant at <a href="https://moffitt.org/">Moffitt Cancer Center</a></b> under supervission of Dr. <a href="http://health.usf.edu/medicine/molecularmedicine/faculty/9903/Blanck">George Blanck</a>.<br>
		        <p style="font-size: 13pt;"><i><b>Pipeline development for identification of systemic mutations (in whole exome sequence (TCGA)) associated with cancer </a> </i></b><br>
		        <p style="font-size: 13pt;"><i><b>Secretome analysis to characterize chronic acidosis of tumor microenvironment</a> </i></b><br>
			 
			 <p style="font-size: 13pt;"><i><b>Proteomics analysis of well differentiated and dedifferentiated liposarcoma</a> </i></b><br>
		<hr>
		
		<tbody><tr>

		<td width="170"><img src="Figures/University_of_South_Florida.png" style="width:80px;height:80"></td>

		<td>
			<p style="font-size: 15pt;"><b>Graduate Research Assistant at <a href="http://labs.cas.usf.edu/cbb/"> 
			Computational Biophysics Lab, University of South Florida</a></b><br>
			<!--Project Name-->
			<!-- <p style="font-size: 13pt;"><b>Graduate Research Assistant at <a href="http://labs.cas.usf.edu/cbb/"> 
			Computational Biophysics Lab, University of South Florida</a></b><br>
			 --><br>
			 
			<p style="font-size: 13pt;"><i><b>Modeling of Dynamic Allostery in Proteins Enabled by Machine Learning</a> </i></b><br>
			<blockquote>Allostery regulation or change of the protein function in a physically distant site from binding site, 
			occurs in many proteins with minor structural difference. There is no method that can fully characterizes 
			this signal transduction phenomenon. By using machine learning methods and graph theory we developed a method 
			that can characterize this mechanism in PDZ2.</blockquote>
			<img src="Figures/pdzNetworks.png"  style="width:800px;height:auto"><br>
			<p style="font-size: 13pt;"><b>Reference:</b>
			<p style="font-size: 11pt;"> <a href="https://aip.scitation.org/doi/abs/10.1063/1.5022469">
			<b>M. Botlani</b>, A. Siddiqui, S. Varma, <i>"Machine Learning Enabled Approach to Incorporate Multi-State 
			Information in Molecular Modeling of Dynamic Allostery:A Case Study of the PDZ2 Domain"</i>. Journal of Cheical Physics (2018)</a> <br>
			
			<p style="font-size: 11pt;"> <a href="https://www.eurekalert.org/pub_releases/2018-04/aiop-mlt041218.php">
			<b>Its Media Coverage <i>"Machine learning techniques may reveal cause-effect relationships in protein dynamics data"</i> eurekalert.org, sciencedaily.com </a> <br>
		<hr>
		
		

			
		
		</td>

	</tr>
	
	
		<tbody><tr>

		<td width="170"><img src="Figures/University_of_South_Florida.png" style="width:80px;height:80"></td>

		<td>

		
		<br>
			<!--Project Name-->
			<!-- <p style="font-size: 13pt;"><b>Graduate Research Assistant at <a href="http://labs.cas.usf.edu/cbb/"> 
			Computational Biophysics Lab, University of South Florida</a></b><br>
			<br> -->
			<p style="font-size: 13pt;"><i><b> Motif co-occurance analysis of Proteomes of S. aureus, B. subtilis and E. coli </a> </i></b><br>
			<blockquote>To see if the protein prenylation (addition of prenyl groups to peptide chains), 
			putatively occurs in prokaryotes we perform a bioinformatic analysis. We searched the S. aureus B. subtilis and E. coli proteomes
			for translated peptides containing the characteristic eukaryotic 'CAAX' motif at their C-terminus. 
			Just few proteins found that contained the prototypical CAAX-motif in  Which suggests in prokaryotes either proteins 
			are not prenylated in bacterial organisms or that such modifications occur at entirely different protein recognition sequences. 
			(C=cysteine, A=any aliphatic amino acid, X=any amino acid) The possibilities for the aliphatic amino acids (A) are: G, A, V, L, I, P, or M.</blockquote>

			<p style="font-size: 13pt;"><b>Reference</b>:
			<p style="font-size: 11pt;"> <a href="http://onlinelibrary.wiley.com/doi/10.1111/mmi.12900/full">
			CN. Krute, RK. Carroll, FE. Rivera, A. Weiss, RM. Young, A. Shilling, <b>M. Botlani</b>, S. Varma, BJ. Baker, LN. Shaw,
			<i>"The Disruption of Prenylation Leads to Pleiotropic Rearrangements in Cellular Behavior in Staphylococcus aureus"</i>, Molecular microbiology(2015)</a> <br>
			
			<hr>
		
		
		<br>
			<!--Project Name-->
			<br>
			<p style="font-size: 13pt;"><i><b>Quantification of Ensemble Changes by a Direct Comparison </a> </i></b><br>
			<blockquote>Direct comparison of conformational ensembles using inverse machine learning to quantify the changes in 
			fluctuations as well as structural changes. This quantification is vital for characterization of 
			regulatory mechanism of proteins with small structural changes due to activation. 
			Many protein families are showing this behavior such as GPCRs, PDZ domains and T-cell receptors. 
			The method uses support vector machine(SVM) mathematical framework for this quantification 
			and showed accuracy in reproducing overlap estimated for a variety of distributions.</blockquote>
			<img src="Figures/ensembleComparison.PNG"  style="width:1000px;height:auto">
		<hr>
		
		
		
			<br>
			<!--Project Name-->
			<!-- <p style="font-size: 13pt;"><b>Graduate Research Assistant at <a href="http://labs.cas.usf.edu/cbb/"> 
			Computational Biophysics Lab, University of South Florida</a></b><br>
			<br> -->
			<p style="font-size: 13pt;"><i><b>Applications of Direct Comparison </a> </i></b><br>
			<blockquote>These applications are all pertaining to characterization of fusion mechanism of Nipah virus. 
			This mechanism starts by attachment of NiV-G protein to the host cell and its binding signal transduction
			to its fusion protein (NiV-F). In the first study by cross-comparison of 3 different ensembles 
			which showed similar function in signal transduction we found a subset of residues that were undergo 
			similar perturbation. We speculate that this intersecting pathway contains signaling pathway as it 
			showed good agreement with experimental data [1]. In the second study we should the effect 
			of force field (implicit vs. explicit water molecules) on estimating changes in perturbation due to binding. 
			The results showed this effect is more pronounced for intersecting pathway [2]. 
			Third study characterizes the effect of mutation on potential signaling pathway and dimeric configuration rearrangement [3].
			(ISCB) 2016 meeting's 3rd best poster depicts most of these studies[4].</blockquote>
			<img src="Figures/svmApplications.png"  style="width:1000px;height:auto">
			<p style="font-size: 13pt;"><b>References:</b>
			<p style="font-size: 11pt;"> <a href="http://onlinelibrary.wiley.com/doi/10.1002/prot.24541/abstract">
			1.	S. Varma, <b>M. Botlani </b>, RE. Leighty,<i> "Discerning intersecting fusion-activation pathways in the Nipah virus using machine learning"</i>
			, Proteins: Structure, Function, and Bioinformatics(2014)</a> <br>
			<p style="font-size: 11pt;"> <a href="http://pubs.acs.org/doi/abs/10.1021/jp5089096">
			2.	P. Dutta, <b>M. Botlani</b>, S. Varma, <i>"Water Dynamics at Protein–Protein Interfaces: Molecular Dynamics Study of Virus-Host Receptor Complexes"</i>
			, The Journal of Physical Chemistry B(2014)</a> <br>
			<p style="font-size: 11pt;"> <a href="http://www.sciencedirect.com/science/article/pii/S0006349516307664">
			3.	P. Dutta, A. Siddiqui, <b>M. Botlani</b>, S. Varma, <i>"Stimulation of Nipah Fusion: Small Intradomain Changes Trigger Extensive Interdomain Rearrangements"</i>
			, Biophysical Journal(2016) </a> <br>
			<p style="font-size: 11pt;"> <a href="ISCB2016_poster_Mohsen.pdf">
			4. <b>M. Botlani </b>, A. Siddiqui, S. Varma, <i>"Quantifying conformational ensemble changes in proteins using inverse machine learning"</i>
			Intelligent Systems for Molecular Biology(ISCB), (2016) meeting. Orlando, USA.
		<hr>	
		
		</td>

	</tr>
	
	
	
			<tbody><tr>

		<td width="170"><img src="Figures/University_of_South_Florida.png" style="width:80px;height:80"></td>

		<td>

			<br>
			<!--Project Name-->
			<!-- <p style="font-size: 13pt;"><b>Graduate Research Assistant at <a href="http://labs.cas.usf.edu/cbb/"> 
			Computational Biophysics Lab, University of South Florida</a></b><br>
			<br> -->
			<p style="font-size: 13pt;"><i><b>Force Field Effects on Simulated D-band Length of Type I Collagen </a> </i></b><br>
			<blockquote>Collagen is the primary constituent of the extracellular matrix of animal connective tissue. 
			Type I is the predominant collagen in mature tendons and ligaments, where it gives them their load-bearing mechanical properties. 
			However, to date, no atomistic simulations of this diffraction model under zero-stress conditions have reported 
			to successfully reproduce the D-band length (captured by TEM). In this study we evaluate systemically the effect of 
			several factors on D-band shrinkage. Based on the results the force field plays a major role in this shrinkage. 
			Generally, force fields whose their backbone dihedral energy landscape matches more closely with our computed CCSD(T) values
			produces a small D-band shrinkage.</blockquote>
			<img src="Figures/collagen.png"  style="width:1000px;height:auto">
			<p style="font-size: 13pt;"><b>Reference:</b>
			<p style="font-size: 11pt;"> <a href="http://onlinelibrary.wiley.com/doi/10.1002/prot.24864/full">
			S. Varma, <b>M. Botlani</b>, JR. Hammond, HL. Scott, JPRO. Orgel, JD. Schieber,
			<i>"Effect of intrinsic and extrinsic factors on the simulated D-band length of type I collagen"</i>,
			Proteins: Structure, Function, and Bioinformatics(2015)</a> <br>
			
		<hr>
		</td>

	</tr>
	
	<tbody><tr>

		<td width="170"><img src="Figures/iums.png" style="width:80px;height:80"></td>

		<td>

			<br>
			<!--Project Name-->
			<p style="font-size: 15pt;"><b>Research Assistant at <a href="http://www.english.mui.ac.ir/content/bioinformatics-research-center">
			Bioinformatics Research Center, Isfahan University of Medical Sciences</a></b><br>
			<br>
			<p style="font-size: 13pt;"><i><b>GPCR (adenosine receptors) molecular docking</a></i> </b><br>
			<blockquote>About 50% of all modern medicines act on G protein-coupled receptors (GPCRs). 
			This study by using molecular docking and subsequent analysis characterizes the receptor-ligand interactions
			of several complexes of A2A adenosine receptors with following PDB IDs: 2YDO, 2YDV, 3EML and 3RFM that bounded to
			Adenosine, NECA, ZM241385 and Caffeine respectively.</blockquote>
			<img src="Figures/gpcrDocking_3qak.png"  style="width:800px;height:auto">
		<hr>
		
		</td>

	</tr>	

	
	<tbody><tr>

		<td width="170"><img src="Figures/iut-logo.png" style="width:80px;height:80"></td>

		<td>

			<br>
			<!--Project Name-->
			<p style="font-size: 15pt;"><b>Scientist at <a href="http://www.iut.ac.ir/en/">Isfahan University of Technology</a> </b><br>
			 
			<p style="font-size: 13pt;"><i><b>A Molecular-Dynamics Study of Freezing Behavior and Structures of Cobalt Nanoclusters</a></i> </b><br>
			<blockquote>By molecular dynamics with general embedded atom method the structures and energies are
			calculated for cobalt nanoclusters with the diameter of 1.5 - 3 nm (147 - 1215 atoms) at two
			different cooling rates. Also to determine the glassy state temperature, the results were
			analyzed by Radial Distribution Function. It was found that the structure of cobalt
			nanocluster in the specified size changes from icosahedral and decahedral for smaller clusters
			to FCC for larger clusters with estimated crossover about 2.55 nm (923 atoms). Moreover,
			the effect of icosahedral magic numbers was observed only for cluster with 147 and 309 atoms.</blockquote>
			<img src="Figures/nanoclusters.jpg"  style="width:1000px;height:auto">
		<hr>	
		</td>

	</tr>
		

	
	
	<tbody><tr>

		<td width="60"><img src="Figures/mobarekeSteelCo.jpg" style="width:100px;height:50"></td>

		<td>

			<br>
			<!--Project Name-->
			<p style="font-size: 15pt;"><b>Data Analyst at <a href="https://en.msc.ir/">Mobarakeh Steel Company(MSC)</a> </b><br>
			
			<p style="font-size: 13pt;"><i><b>Data Modelling of Hot Mill Steel Products</a> </i></b><br>
			<blockquote>In this study we used annual product of Mobarakeh Steel Company (MSC) the largest steel maker of MENA (Middle East & Northern Africa) 
			region to model, characterize and predict the mechanical properties of the products. 
			Bayesian Regularized Neural Network (BRNN) were used to make a regression model (with 70234 samples) 
			for predicting tensile strength of products [1, 2]. The relative importance of the input variables 
			also were estimated by a sensitivity analysis. A Bayesian Neural Network classifier, with uses Reversible jump Markov Chain Monte Carlo
			to find right complexity of the model were developed for predicting steel grain size based on input parameters of 
			hot mill(with 624 grain size images)[3].</blockquote>
            The full study is available in this book chapter[4] (with more than 8100 downloads)
			<br>
			<img src="Figures/steelMill.png"  style="width:1000px;height:auto">
			<br>
			<p style="font-size: 13pt;"><b>References:</b>
			<p style="font-size: 11pt;"> <a href="http://www.sciencedirect.com/science/article/pii/S0261306909000818">
			1.	<b>M. Botlani</b>, MR Toroghinejad, AR Key Yeganeh<i>"Modeling the Yield Strength of Hot Strip Low Carbon Steels by Artificial Neural Network"</i>, Materials & Design(2009)</a> <br>
			<p style="font-size: 11pt;"> <a href="https://www.jstage.jst.go.jp/article/isijinternational/49/10/49_10_1583/_article">
			2.	<b>M. Botlani</b>, MR Toroghinejad, S Abbasi,<i> "Artificial Neural Network Modeling the Tensile Strength of Hot Strip Mill Products</i>, The Iron and Steel Institute of Japan(ISIJ)(2009)</a><br>
			<p style="font-size: 11pt;"> <a href="http://www.doiserbia.nb.rs/Article.aspx?id=0352-51391200011B">
			3.	<b>M. Botlani</b>, MR Toroghinejad, <i>"Application of Bayesian ANN and RJMCMC to predict the grain size of hot strip low carbon</i>, Journal of the Serbian Chemical Society(2012) </a> <br>
			<p style="font-size: 11pt;"> <a href="http://cdn.intechweb.org/pdfs/14733.pdf">
			4.	MR Toroghinejad, <b>M. Botlani</b>, <i>"Application of Bayesian Neural Networks to Predict Strength and Grain Size of Hot Strip Low Carbon Steels</i>, INTECH Open Access Publisher(2011)</a> <br>
		<hr>	
			
		</td>

	</tr>
	
	
	
	
	<tr><td>&nbsp;</td></tr></tbody></table>

			 
			 
   			 <hr>    
   		     <h2> Teaching </h2>
<!--    		     <b> - Instructor</b>:
			 <ul>
                <li> CDA3201: Computer Logic Design - Summer 2013.
                <li> CDA3201L: Computer Logic Design Lab - Summer 2014 [<a href="http://www.eng.usf.edu/~chakeri/Evaluation.pdf"> Evaluations </a>] </li>             
            </ul>
 -->			<b> - Teaching Assistant</b>:
			<ul>
				<li> Biology of Human, Biology I cellular processes and General Genetics</li> 
			 </ul>
   			 <hr> 			     					 
   		     <h2> Publications  - <a href = "https://scholar.google.com/citations?user=vxA3-f0AAAAJ&hl=en" target="_blank"> (<small>Google Scholar</small>) </a> </h2>
   			 
			 <h2> Codes  - <a href = "https://github.com/mohbot" target="_blank"> (<small>GitHub</small>) </a> </h2>
			 
			 
   			 <br>
   			 <br>
	
	<tr><td>&nbsp;</td></tr></tbody></table>
				 
			 <hr>    
   		     <h2> Personal Interests </h2>
   		     <ul>
                <i><p style="font-size: 13pt;">"Playing Guitar and Bicking".</i> 
				<br>
				<br>


		</td>

	</tr>
	<!--<tr><td>&nbsp;</td></tr></tbody></table>
				
				
            </ul>
			
			<a href="http://www.hitwebcounter.com" target="_blank">
			<div style="text-align: center;">
			<img src="http://hitwebcounter.com/counter/counter.php?page=6203179&style=0025&nbdigits=4&type=ip&initCount=0" title="" Alt=""   border="0" style="width:40px;height:20">
			</div>
			</a>-->
                            
		  
</body>
</html>
